testing/libspatialite/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: Bradley J Chambers <brad.chambers@gmail.com>
+# Maintainer: Bradley J Chambers <brad.chambers@gmail.com>
+pkgname=libspatialite
+pkgver=4.3.0
+_pkgver="$pkgver"a
+pkgrel=0
+pkgdesc="SpatiaLite is an open source library intended to extend the SQLite core to support fully fledged Spatial SQL capabilities."
+url="https://www.gaia-gis.it/fossil/libspatialite/index"
+arch="all"
+license="MPLv1.1"
+options="!check"
+makedepends="automake geos-dev sqlite-dev proj4-dev zlib-dev libxml2-dev"
```

Comment:
Is `automake` required? You don't seem to regenerate the autoconfig things, so this might be not needed.

Reply:
No. Maybe a copy/paste error. Will fix.